.github/workflows/release-canary.yml

```diff
@@ -27,18 +27,25 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Setup
-        uses: ./.github/actions/setup
+        id: config
+        uses: ./.github/actions/init
+        with:
+          turbo-signature: ${{ secrets.TURBO_REMOTE_CACHE_SIGNATURE_KEY }}
+          turbo-team: ${{ vars.TURBO_TEAM }}
+          turbo-token: ${{ secrets.TURBO_TOKEN }}
+          playwright-enabled: true # Must be present to enable caching on branched workflows
 
       - name: Version packages for canary
         id: version-packages
         run: npm run version:canary | tail -1 >> "$GITHUB_OUTPUT"
 
       - name: Canary release
         if: steps.version-packages.outputs.success == '1'
-        run: npm run release:canary
+        run: npx turbo build $TURBO_ARGS && changeset publish --tag canary --no-git-tag
```

Comment:
❓ Are we not using `release:canary` now so we can use `$TURBO_ARGS`?

Reply:
That's correct.

Reply:
I completely get why but I'm a bit skeptical about this change as we now have this script in 2 different places (across 2 different active branches, main and release/v4) and its very likely that these will diverge accidentally. Shall we turn them into node/shell scripts so we can pass args directly to `turbo build` without the need for duplication?